ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayLayerViewDrawStatus/DisplayLayerViewDrawStatus.qml

```diff
@@ -15,112 +15,58 @@
-        rowDelegate: Row {
-            height: tableView.height / 3
-        }
 
-        // create rectangle to frame the TableView
-        style: TableViewStyle {
-            backgroundColor: "transparent"
-            frame: Rectangle {
-                border.color: "black"
-                radius: 10
-
-                // make sure mouse actions on table do not affect map behind it
-                MouseArea {
-                    anchors.fill: parent
-                    onClicked: mouse.accepted = true
-                    onWheel: wheel.accepted = true
-                }
+        Rectangle {
+            anchors {
+                bottom: mapView.attributionTop
+                horizontalCenter: parent.horizontalCenter
+                margins: 5
             }
-        }
-
-        // create List Model to store Layer View States and names
-        ListModel {
-            id: layerViewModel
-        }
-
-        TableViewColumn {
-            role: "name"
-            width: tableView.width * 0.75 - tableView.anchors.margins
-            delegate: Component {
-                Text {
-                    text: styleData.value
-                    leftPadding: tableView.anchors.margins
-                    renderType: Text.NativeRendering
-                    horizontalAlignment: Text.AlignLeft
-                    verticalAlignment: Text.AlignVCenter
-                    elide: Text.ElideRight
-                    font {
-                        weight: Font.Black
-                        pixelSize: tableView.height * 0.10
+            border.color: "black"
+            radius: 10
+            height: childrenRect.height
+            width: childrenRect.width
+            opacity: 0.95
+            GridLayout
+            {
+                id: gridLayout
+                flow: GridLayout.TopToBottom
+                Connections {
+                    target: statusModel
+                    onRowsInserted: gridLayout.rows = statusModel.rowCount();
+                }
+                Repeater {
+                    model: statusModel
+                    delegate: Text {
+                        text: name
+                        horizontalAlignment: Text.AlignLeft
+                        Layout.margins: 5
                     }
                 }
-            }
-        }
-
-        TableViewColumn {
-            role: "status"
-            width: tableView.width * 0.25
-            delegate: Component {
-                Text {
-                    text: styleData.value
-                    renderType: Text.NativeRendering
-                    horizontalAlignment: Text.AlignRight
-                    verticalAlignment: Text.AlignVCenter
-                    font.pixelSize: tableView.height * 0.10
-                    color: "steelblue"
+                Repeater {
+                    model: statusModel
+                    delegate: Text {
+                        text: status
+                        color: "steelblue"
```

Comment:
maybe this color should come from the style?

Reply:
Stylecolor is ignored for normal text. Is that what you meant? Sorry, unclear.